CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog
 
+#### WIP
+- Added support for passing ecto `dynamic` filters
+- Added support for passing functions to use for custom filtering
+- Added `EctoShorts.filter` as a shortcut to `CommonFilters.convert_params_to_filter` 
+- Added a default order_by field and direction for queries
+- Added `:id` to common filters
```

Comment:
This shouldn't be necessary, `:id` if a valid field on the schema will get run through the schema filters and be put in